2021-01-09 16:31:57,726 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 16:39:16,902 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 16:41:25,104 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 16:41:49,808 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 16:42:02,028 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 16:42:14,606 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 16:43:14,800 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:02:05,922 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:19:56,260 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:26:28,289 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:26:35,420 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:34:59,420 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:37:24,993 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:40:39,372 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:40:48,986 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:40:57,313 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:47:06,134 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:48:16,602 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:50:58,283 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:55:12,458 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:58:11,880 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:59:23,876 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 17:59:32,684 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 18:01:06,877 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 18:04:00,741 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 18:10:29,500 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 18:15:24,488 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 18:18:00,783 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 19:35:20,051 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 19:41:54,208 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:43]
2021-01-09 19:57:37,860 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:45]
2021-01-09 19:57:59,822 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:45]
2021-01-09 20:00:46,579 ERROR: Exception on /reset_password_request [POST] [in c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\fred9\STP2020\project\app\routes.py", line 103, in reset_password_request
    send_password_reset_email(user)
  File "C:\Users\fred9\STP2020\project\app\email.py", line 20, in send_password_reset_email
    token = user.get_reset_password_token()
  File "C:\Users\fred9\STP2020\project\app\models.py", line 65, in get_reset_password_token
    return jwt.encode(
AttributeError: 'str' object has no attribute 'decode'
2021-01-09 20:02:20,375 ERROR: Exception on /reset_password_request [POST] [in c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\fred9\stp2020\project\env\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\fred9\STP2020\project\app\routes.py", line 103, in reset_password_request
    send_password_reset_email(user)
  File "C:\Users\fred9\STP2020\project\app\email.py", line 20, in send_password_reset_email
    token = user.get_reset_password_token()
  File "C:\Users\fred9\STP2020\project\app\models.py", line 65, in get_reset_password_token
    return jwt.encode(
AttributeError: 'str' object has no attribute 'decode'
2021-01-09 20:07:28,441 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:45]
2021-01-09 20:08:32,825 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:45]
2021-01-09 20:34:38,722 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:47]
2021-01-09 20:37:11,892 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:47]
2021-01-09 20:45:43,257 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:47]
2021-01-09 20:53:13,856 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:49]
2021-01-11 00:28:15,095 INFO: Microblog startup [in C:\Users\fred9\STP2020\project\app\__init__.py:49]
